app/src/tests/components/contents/packets/PacketDetails.test.tsx

```diff
@@ -64,7 +64,7 @@ describe("packet details component", () => {
     });
   });
 
-  it("should not render parameters or files when none", async () => {
+  it("should not render parameters, dependencies or files when none", async () => {
```

Comment:
Can you add a test for this component for when there _are_ dependencies? That would check the integration of PacketDependencies with PacketDetails, i.e., does PacketDetails give PacketDependencies the correct value for the `dependencies` parameter.